src/Plugin/AddPathPlugin.php

```diff
@@ -19,9 +20,18 @@
     private $uri;
 
     /**
+     * @var bool
+     */
+    private $smart;
+
+    /**
      * @param UriInterface $uri
+     * @param array        $config {
+     *
+     *     @var bool $smart True will prepend path only if request path does not start with path to add.
```

Comment:
i am somehow confused by the wording. maybe "Set to true to only prepend the path only if the request path does not start with that path already."?